test/mason/publish/create-registry/SKIPIF

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
This works, but for consistency, can we make this a symlink to the parent SKIPIF like other tests?